index.html

```diff
@@ -31,6 +31,7 @@
   <body>
     <header>
       <a href="/" class="home-btn">Where in the world?</a>
+
       <div class="contrast-button-container">
         <button class="contrast-button">
           <i class="fa-regular fa-moon"></i>
```

Comment:
Zie hieronder Line 44:

Waarom zitten de elementen eigenlijk in een form?

Reply:
Ik denk ook dat de indeling niet lekker responsive werkt.

Ik zou de search container als een stuk doen, en dan de filter en tile-btn-container onder een container. 
Dan kan je het makkelijker spacen. 

nu is dit stuk 3 delen, en als je de scherm groter/kleiner maakt, dan schuift je filter dropdown naar ongemakkelijke plekken.

Reply:
In eerste instantie omdat ik dacht dat elementen als input en select dropdowns in een form geplaatst moeten worden, maar er wordt natuurlijk niks gesubmit dus die heeft geen functie nu?